Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM docker.io/centos:centos7
 
 RUN yum install -y python-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/5609d2e3aee35578e914bcbfac92a46c759c0a31/tripleo_repos/main.py | python - current && \
+    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - current && \
```

Comment:
maybe it's time to add `-b stein`? or do we plan to keep using master?

Reply:
Not sure,I guess we need to keep at master if we want to start using the mdns work (or any other new features) when they are ready. 

Either way this patch is just reversing a workaround we needed at one stage,